novu-workflows/react/translation/app/novu/workflows/translation/workflow.ts

```diff
@@ -0,0 +1,59 @@
+import { workflow } from '@novu/framework';
+import { renderEmail } from '../../emails/welcome-email';
+import i18next from 'i18next';
+import { z } from 'zod';
+
+i18next.init({
+  resources: {
+    en: {
+      translation: {
+        welcomeEmailSubject: 'Welcome to Twitch, {{username}}!',
+        welcomeEmailBody:
```

Comment:
I would suggest the following renames to avoid the `2` suffix on the last body:

* `welcomeEmailBody` -> `welcomeEmailIntroduction`
* `welcomeEmailBody2` -> `welcomeEmailConclusion`